lib/stream/activity.rb

```diff
@@ -15,4 +15,8 @@ def title
   def posts
     @posts ||= EvilQuery::Participation.new(user).posts
   end
-end
\ No newline at end of file
+
+  def contacts_title
+    I18n.translate('streams.activity.contacts_title')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.